inputfiles/addedTypes.json

```diff
@@ -1154,6 +1189,13 @@
                     }
                 ],
                 "overrideType": "ReadableStreamDefaultReader<T>"
+            },
+            {
+                "name": "EventListenerOrEventListenerObject",
+                "type": [
+                    { "type": "_EventListener" },
```

Comment:
Hey, there is `override-type` to workaround this 😄

Reply:
Good point! Rebased this on main and fix that up